### PR TITLE
[DOCS] Adding bootstrapping to README file

### PR DESCRIPTION
**Pull request by DLotero**
## Summary

-Added the bootstrapping kick off guide for the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,17 @@ The boilerplate includes
...
 
+### Bootstrap Kick off
+
+For bootstraping your own project using the template generator script you'll need to follow these steps:
+
+1. run `yo yourPathToWolmo/generators/app/index.js -v` you'll need [Yeoman](https://yeoman.io/learning/index.html) installed for this.
+2. Some prompts will pop up on your terminal. Pick whatever configuration works best for your current proyect.
+3. After the script is finished, your project folder will be successfully generated with all the necessary npm dependencies installed. Do `cd you/project/path`.
+4. It's possible that you'll need to link React Native's libraries using `react-native link` (for push notifications to work for instance).
+5. Run `react-native run-ios` or/and `react-native run-android`.
```
Comment (sfernandez11) [inline README.md]: Before running `react-native run-ios` or `react-native run-android` the bundler must be started with `npm start` or `yarn start`, I think you forgot add this step.

Comment (guidoprinc) [reply README.md]: @DLotero make this change and then go!

File: README.md [hunk 2]
```diff
@@ -54,6 +54,18 @@ The boilerplate includes
 - [Prettier](https://github.com/prettier/prettier)
 - [Reactotron](https://github.com/infinitered/reactotron)
 
+### Bootstrap Kick off
+
+For bootstraping your own project using the template generator script you'll need to follow these steps:
+
+1. run `yo yourPathToWolmo/generators/app/index.js -v` you'll need [Yeoman](https://yeoman.io/learning/index.html) installed for this.
```
Comment (WFolini) [inline README.md]: I'd remove the `-v` arg and add a point clarifying that we can see the logger using `-v`

File: README.md [hunk 3]
```diff
@@ -54,6 +54,18 @@ The boilerplate includes
...
+2. Some prompts will pop up on your terminal. Pick whatever configuration works best for your current proyect.
+3. After the script is finished, your project folder will be successfully generated with all the necessary npm dependencies installed. Do `cd you/project/path`.
+4. It's possible that you'll need to link React Native's libraries using `react-native link` (for push notifications to work for instance).
```
Comment (WFolini) [inline README.md]: I have a doubt about this. In the new v 0.60 we have auto-link feature and maybe we don't need anymore this check

File: README.md (hunk 2 above)
Comment (mcavo) [inline README.md]: The first step should be `npm install` or `yarn install

File: README.md (hunk 3 above)
Comment (DLotero) [reply README.md]: Maybe when we migrate to the new version, we can review this

File: README.md [hunk 4]
```diff
@@ -58,13 +58,14 @@ The boilerplate includes
...
-3. After the script is finished, your project folder will be successfully generated with all the necessary npm dependencies installed. Do `cd you/project/path`.
-4. It's possible that you'll need to link React Native's libraries using `react-native link` (for push notifications to work for instance).
-5. Start budler using `npm run start` or `npm start`.
-6. Run `react-native run-ios` or/and `react-native run-android`.
-7. Start working on your project!
+1. Run `npm install` or `yarn install`.
+2. run `yo yourPathToWolmo/generators/app/index.js` you'll need [Yeoman](https://yeoman.io/learning/index.html) installed for this. Also, the argument `-v` can be used for logging.
+3. Some prompts will pop up on your terminal. Pick whatever configuration works best for your current proyect.
+4. After the script is finished, your project folder will be successfully generated with all the necessary npm dependencies installed. Do `cd you/project/path`.
+5. It's possible that you'll need to link React Native's libraries using `react-native link` (for push notifications to work for instance).
```
Comment (WFolini) [inline README.md]: I'd delete this point because the bootstrap does react-native link itself, and in the 0.60 it is automatic

**Pull request merged**